kiwi/builder/filesystem.py

```diff
@@ -128,13 +129,14 @@ def create(self) -> Result:
         )
         if self.bundle_format:
             self.result.add_bundle_format(self.bundle_format)
+        compression = self.runtime_config.get_bundle_compression(default=True)
+        if self.luks is not None:
+            compression = False
```

Comment:
shouldn't the setting of an eventual runtime config not always have the highest priority ? iirc if luks is activated the compression is switched off even if the runtime config would configure something different. I think this would be unexpected ?